packages/an-anime-game-launcher/an-anime-game-launcher.pacscript

```diff
@@ -0,0 +1,36 @@
+name=an-anime-game-launcher
+version=2.0.0
+description="An Launcher for a specific anime game written in Electron with auto-patching and playtime tracking function"
+url="https://kaifa.ch/an_anime_game_launcher.tar.gz"
+hash="63942d038a0b7a58991e1bcc7b458341de58d7a26e6016153001cbf3ad6d77a2"
```

Comment:
That link will always point to the latest file right? Then the `hash` would be invalid automatically when the link updates. Point to something static.

Reply:
the link/file will only update when the pacscript has been updated since I control the tar file